src/rpc/blockchain.cpp

```diff
@@ -2788,6 +2788,66 @@ static RPCHelpMan loadtxoutset()
     };
 }
 
+static RPCHelpMan monitorsnapshot()
+{
+return RPCHelpMan{
+        "monitorsnapshot",
+        "\nReturn information about UTXO snapshot status.\n",
```

Comment:
It would be useful to have information about dbcache allocation (and usage) here.

Reply:
Done!